src/main/java/com/iota/iri/crypto/ISS.java

```diff
@@ -46,6 +86,17 @@
         return subseed;
     }
 
+    /**
+     * Derives a secret key of length {@code FRAGMENT_LENGTH * numberOfFragments} from subseed. <br>
```

Comment:
{@value #FRAGMENT_LENGTH} * {@code numberOfFragments}

This will enable the reader to see what is `FRAGMENT_LENGTH`.
Another option is to use {@link} to make the value discovery one click away.